convert_v2.py

```diff
@@ -0,0 +1,110 @@
+import sys
+import argparse
+import gzip
+import shutil
+import tarfile
+import rarfile
+import patoolib
+import unrar
+
+class FilesFamily:
+
+    def __init__(self, list_of_formats):
+        """
+        This is the constructor function.
```

Comment:
This comment is redundant, documenting what the purpose of list_of_formats isn't bad thought